templates/services/jobs/scheduled/manifest.yml

```diff
@@ -0,0 +1,38 @@
+# The manifest for the "{{.Name}}" job.
+# Read the full specification for the "{{.Type}}" type at:
+#  https://github.com/aws/copilot-cli/wiki/Manifests#scheduled-job
+
+# Your job name will be used in naming your resources like log groups, ECS Tasks, etc.
+name: {{.Name}}
+# The "architecture" of the job you're running.
+type: {{.Type}}
+
+image:
+  # Docker build arguments. You can specify additional overrides here. Supported: dockerfile, context, args
+  build: {{.Image.Build.BuildArgs.Dockerfile}}
+
+# Number of CPU units for the task.
+cpu: {{.CPU}}
+# Amount of memory in MiB used by the task.
+memory: {{.Memory}}
+
+# The number of times to retry the job before failing.
+retries: 1
+# The timeout after which to stop the job if it's still running. 
+timeout: 900
+# The trigger for your job. You can set the `rate` or `cron` and `eventpattern` fields.  
```

Comment:
nit: should we remove the `eventpattern` from the comment, it'll probably be added in a further milestone

Reply:
Reworked this to simply include "schedule." We can add event patterns as a new job architecture with a different manifest parsing pattern in the future.